client/src/components/WarningText.js

```diff
@@ -21,7 +22,11 @@ export const WarningText = ({
       margin={{ top: 'small', bottom: 'medium' }}
     >
       <Box margin={iconMargin} pad={iconPad}>
-        <Icon color={iconColor} size={iconSize} name="Warning" />
+        <Icon
+          color={iconColor}
+          size={iconSize}
+          name={`Warning${iconNoFill ? 'NoFill' : ''}`}
```

Comment:
Can we just have a prop called `iconName` and default it to `WarningNoFill`. Would provide more flexibility and I'm pretty sure we will always use the "no fill" option.

Reply:
Ah ok, So in that case we should just switch to `iconName` and default it to `Warning` and then for this one use case we pass in `WarningNoFilll`. We won't have to update the other uses of this component and we can now expand it as we need for new iconography.